grails-app/domain/org/transmartproject/db/dataquery/highdim/tworegion/DeTwoRegionJunction.groovy

```diff
@@ -0,0 +1,71 @@
+package org.transmartproject.db.dataquery.highdim.tworegion
+
+import groovy.transform.EqualsAndHashCode
+import org.transmartproject.core.dataquery.highdim.tworegion.Junction
+import org.transmartproject.db.dataquery.highdim.DeSubjectSampleMapping
+
+/**
+ * Created by j.hudecek on 4-12-2014.
+ */
+@EqualsAndHashCode()
+class DeTwoRegionJunction implements Serializable, Junction {
+
+    Long upEnd
+    String upChromosome
+    Long upPos
+    Character upStrand
+    Long downEnd
+    String downChromosome
+    Long downPos
+    Character downStrand
+    Boolean isInFrame
+    Long id
+    DeSubjectSampleMapping assay
+
+    Boolean isInFrame() {
+        return isInFrame
+    }
```

Comment:
OK, what I said was not correct. The generated getter will be `getIsInFrame` because the type is `Boolean`, not `boolean`. So this method is actually needed. BTW: you should drop the return keyword when it's not necessary.